doc/protocol/protocol_4.md

```diff
@@ -0,0 +1,1062 @@
+Network protocol (version 1)
```

Comment:
Honestly, I think we should just rename this to protocol.md and not have per-version documentation. We're storing the history of changes to this document in git and by doing this renaming thing we remove the ability to easily use git to trace back the history of the document.

Reply:
Agreed, that is simpler I'll change it.